scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/data/TaskProActiveDataspaces.java

```diff
@@ -202,6 +211,30 @@ private void initDataSpaces() throws Exception {
             logger.warn("Missing permission to change write permissions to " + getScratchFolder());
         }
 
+        InputOutputSpaceConfiguration cacheConfiguration = DataSpaceNodeConfigurationAgent.getCacheSpaceConfiguration();
+
+        if (cacheConfiguration != null) {
+            final String cacheName = cacheConfiguration.getName();
+
+            cacheSpaceInstanceInfo = new SpaceInstanceInfo(appId, cacheConfiguration);
+            try {
+                namingService.register(cacheSpaceInstanceInfo);
+            } catch (SpaceAlreadyRegisteredException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1166)